src/diffusers/models/normalization.py

```diff
@@ -266,6 +263,7 @@ def forward(
         hidden_dtype: Optional[torch.dtype] = None,
     ) -> Tuple[torch.Tensor, torch.Tensor, torch.Tensor, torch.Tensor, torch.Tensor]:
         # No modulation happening here.
+        added_cond_kwargs = added_cond_kwargs or {"resolution": None, "aspect_ratio": None}
```

Comment:
oh what is this? is this fixing a current bug?

Reply:
Yep. We accept `None` in `added_cond_kwargs` here, but we actually need to pass values for `resolution` and `aspect_ratio` in the following pixart embedding layer (which requires it as non-defaulted arg)